README.md

```diff
@@ -20,81 +20,47 @@ $ aws-okta add
 
-### Configuring your aws config
-
-`aws-okta` assumes that your base role is one that has been configured for Okta's SAML integration by your Okta admin. Okta provides a guide for setting up that integration [here](https://support.okta.com/help/servlet/fileField?retURL=%2Fhelp%2Farticles%2FKnowledge_Article%2FAmazon-Web-Services-and-Okta-Integration-Guide&entityId=ka0F0000000MeyyIAC&field=File_Attachment__Body__s).  During that configuration, your admin should be able to grab the AWS App Embed URL from the General tab of the AWS application in your Okta org.  You will need to set that value in your `~/.aws/config` file, for example:
+The most seamless way to use `aws-okta` is via AWS SDK config credential
+process. To do so you will need to setup your AWS config as `aws-okta` assumes
+that your base role is one that has been configured for Okta's SAML integration
+by your Okta admin. Okta provides a guide for setting up that integration
+[here](https://support.okta.com/help/servlet/fileField?retURL=%2Fhelp%2Farticles%2FKnowledge_Article%2FAmazon-Web-Services-and-Okta-Integration-Guide&entityId=ka0F0000000MeyyIAC&field=File_Attachment__Body__s).
+During that configuration, your admin should be able to grab the AWS App Embed
+URL from the General tab of the AWS application in your Okta org.  You will need
+to set that value in your `~/.aws/config` file, for example:
 
 ```ini
-[okta]
+[profile okta]
 aws_saml_url = home/amazon_aws/0ac4qfegf372HSvKF6a3/965
+okta_aws_role_arn = arn:aws:iam::<account-id>:role/<okta-role-name>
+credential_process = aws-okta cred-process okta
 ```
 
-Next, you need to set up your base Okta role.  This will be one your admin created while setting up the integration.  It should be specified like any other aws profile:
-
-```ini
-[profile okta-dev]
-role_arn = arn:aws:iam::<account-id>:role/<okta-role-name>
-region = <region>
-```
-
-Your setup may require additional roles to be configured if your admin has set up a more complicated role scheme like cross account roles.  For more details on the authentication process, see the internals section.
+Your setup may require additional roles to be configured if your admin has set
+up a more complicated role scheme like cross account roles.  For more details on
+the authentication process, see the internals section.
 
 #### A more complex example
 
-The `aws_saml_url` can be set in the "okta" ini section, or on a per profile basis. This is useful if, for example, your organization has several Okta Apps (i.e. one for dev/qa and one for prod, or one for internal use and one for integrations with third party providers). For example:
+The `aws_saml_url` can be set in the "okta" ini section, or on a per profile
+basis. This is useful if, for example, your organization has several Okta Apps
+(i.e. one for dev/qa and one for prod, or one for internal use and one for
+integrations with third party providers). For example:
 
 ```ini
-[okta]
+[profile okta-dev]
```

Comment:
ditto here

Reply:
Yeah, I think I will need to give it a try specifying the [okta] part... Maybe this change isn't needed then 🤔